### ISSUE TITLE
explain currently only supports English

### ISSUE THREAD

**Issue by Richie94** [amaiya/ktrain#26]
Hey there,
at first thanks for providing such an awesome framework, it's nice to work with. I'll try to get some insights about what my model has learned, but I do work on german data. So I wanted to know, how it is possible, to fit the current text-explainer to also handle german or other languages, because I do not see why that only currently works for english.
Best,
Richard

**Comment by amaiya** [comment]
@Richie94   v0.6.0 is being released soon, and I'll try to enable this for non-English.  Thanks.

**Comment by Richie94** [comment]
thats nice, but I also wanted to  understand, why the current approach does not support english. As I understand the eli5 TextExplainer LIME approach is model agnostic, so does not care about anything but just sampling input data and checking the prediction change...

**Comment by amaiya** [comment]
You're understanding is correct.  I believe `explain`  was originally disabled (incorrectly) because languages like Chinese were not being tokenized properly within `explain` prior to invoking eli5.  This is now fixed in v0.6.1 (instead of 0.6.0 as I indicated before).   So, you should now be able to use `explain` with non-English text if you upgrade to v0.6.1. Thanks.
